src/Components/AddTodo/AddTodo.js

```diff
@@ -0,0 +1,111 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export class AddTodo extends Component {
+  state = {
+    title: '',
+    errorInput: false,
+    errorSelect: false,
+    userId: 0,
+  }
+
+  handleSelectChange = ({ target: { value } }) => {
```

Comment:
think about how to combine this func and the next one

Reply:
when i try to do that, i have problems with value

Reply:
you could check if it's a number